book/src/background/groups.md

```diff
@@ -36,14 +36,14 @@ Assuming the discrete log assumption holds, Pedersen commitments are also perfec
 and computationally binding:
 
 * **hiding**: the adversary chooses messages $m_0, m_1.$ The committer commits to one of
-  these messages $c = \text{Commit}(m_b;r), b \in \{0,1\}.$ Given $c,$ the probability of
+  these messages $c = \text{Commit}(m_b,r), b \in \{0,1\}.$ Given $c,$ the probability of
```

Comment:
This notation is inconsistent with the protocol spec, which uses Commit(r, m) or equivalently Commit<sub>r</sub>(m).